wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
+Under normal use with unmodified software, your user profile information is exchanged with Servers you connect to, peers connected to those same Servers, and to third parties (including Directories) that use the Jamulus protocol. This information is limited to your Jamulus name, city, country, instrument, and skill level as you have set them in your profile. The Servers you connect to will also have access to your internet address (IP Address) as it is required for the software to work; but this information is not shared with peers on the same Server or available to third parties. 
 
-### Use of IP Addresses
+All communications between and among Clients, Servers, Directories, and third-party protocol users are sent without encryption.
 
-When you connect to Server, the Server operator can see your IP address while you are connected.  If the Server operator has enabled logging (which is off by default) your IP address will also be logged and stored in the Server's log file.
+## Jamulus Servers
 
-The IP addresses of all Servers registered with the Directory can also be seen by third parties for informational or other purposes (for example [here](https://explorer.jamulus.io/)). Your public IP address is otherwise not logged or stored by Jamulus, but may be stored or processed by third parties.
+When you connect to a Server, either directly or through a Directory, the operator of that Server is responsible for its operation policy, privacy policy, and data use policy. While unmodified Servers do not log or store your connection or profile information, some modified Servers may do this; therefore, you should have no expectation of privacy with respect to your profile information or internet address.
 
-### Audio Recordings
+## Chat Exchanges
 
-You will see a notice if you are connected to a Jamulus Server when Server recording is turned on. Recordings of each player are stored by the Server separately as .WAV files and only the Server operator has access to them unless they choose to make them available to third parties.
+Chats are textual messages that can be exchanged between Clients connected to the same Server. Everyone connected to a Server can see all chats and there should be no expectation of privacy with respect to information sent through the chat feature of Jamulus. While unmodified Servers do not log or store chats, some modified Servers may do this.
 
-### Text Chat
+## Audio Recordings
 
-When you type a message in the Chat Window, other connected players can see that, but chats are not stored by the Server and neither the Server operator nor any third parties have access to them.
+Unmodified Servers will display a notice if recording is turned on. Recordings of each channel are stored by the Server and are controlled by the Server operator.
```

Comment:
```suggestion
Unmodified Servers will display a notice if recording is turned on. Recordings of each connection are stored by the Server and are controlled by the Server operator.```

It's a connection I guess.

Reply:
The suggestion syntax here is wrong I think.

Reply:
A stereo connection has two channels. Yes?

Reply:
> (I think you can can connect to a Server without creating a channel?)

No - the server doesn't recognise a "connection" without audio being sent.  (UDP is connectionless.)

Reply:
> No - the server doesn't recognise a "connection" without audio being sent. (UDP is connectionless.)

Oh OK. I was thinking of the case where a server uses `-L` and the participant hasn't accepted yet.

Reply:
As far as I remember, the client isn't recognised (and the response indicating the prompt should be displayed isn't sent) until the first audio packet is received.  I don't remember how the mix is handled (you can hear others but not be heard).